app/controllers/teams_controller.rb

```diff
@@ -97,8 +98,64 @@ def destroy
     end
   end
 
-  private
+  def match_mentor
+    @team = Team.find(params[:id]) || (record_not_found && return)
+    !authenticate_user(true, false,
+                       @team.get_relevant_users(true, true)) && return
+    cohort = @team.cohort || current_cohort
+    @teamsMentorMatchings = MentorMatchings.where(:team_id => params[:id]);
+    
+    render locals: {
+      mentors: Mentor.joins(:user).select('users.*').where(cohort: cohort),
```

Comment:
This causes the `id` displayed for mentor to be actually the `user_id`. It works well if the `user_id` is the same as `id` of the mentor model. But if they are different it will cause a bug. But I'm also not really sure if that is possible for the mentor id to be different than the mentor's user_id.